components/nimbus/src/behavior.rs

```diff
@@ -129,6 +129,14 @@ impl IntervalData {
             starting_instant: Utc::now(),
         };
         data.buckets.push_front(0);
+        data.starting_instant = data
```

Comment:
Can we get a brief comment here explaining what's going on here and why?

Reply:
Yep!